.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Base owners : Benjamin Rotendahl & Kristoffer Rath Hansen
+@rotendahl @lakridserne
+
+# If you want to own a code section, let us know. 
```

Comment:
Så er vi basis code owners andre kan tilføjes med f.eks /template @hans hvis vi ønsker at en anden skal have ansvar for den del af koden